lib/noted/notes.ex

```diff
@@ -369,7 +385,15 @@ defmodule Noted.Notes do
         error ->
           error
       end
-    end
+    end)
+    tags
+  else
+    :ok
+  end
```

Comment:
This looks like the formatter needs to be run :)

Reply:
ran the formatter https://github.com/lawik/noted/pull/2/commits/d0e9f1146bc594b3e659b2bf2c96499d7828854e